test/controllers/LandingPageControllerSpec.scala

```diff
@@ -107,7 +107,7 @@ class LandingPageControllerSpec extends ItSpec {
 
       paragraphs(0)
         .text() shouldBe "Use this service to set up a payment plan for your outstanding employers’ PAYE bill. " +
-        "Payments are taken by Direct Debit and include interest charged at the Bank of England base rate plus 2.5% per year."
+        "Payments are taken by Direct Debit and include interest charged at the Bank of England base rate plus 4.0% per year."
```

Comment:
shouldn't this just say `4%` instead of `4.0%`? You may need to do some formatting if it's a whole number to make that happen

Reply:
it should do, I think, but im unsure what I need to do to make it do that

Reply:
it could be something as simple as this maybe:
```scala
def formatPercentage(d: BigDecimal): String = d.toString.stripSuffix(".0")
```